static/include/program/authenticate.md

```diff
@@ -0,0 +1,118 @@
+To authenticate yourself to your robot, you need
+
+1. The robot part's api key:
+
+   To authenticate, [use a robot part API key](/manage/robots/#security) or [an organization-wide API key](/manage/cli/#create-an-organization-api-key).
```

Comment:
Is this what we want to recommend to users?

Reply:
we will be releasing the ability to create API keys across all types of resources and combinations soon (i.e an api key can have an authorization on a org, location, robot or any combination of all three). this is correct for now though but it will be changing shortly. up to you if you want to make it more generic now or later.